drivers/usb/udc/udc_stm32.c

```diff
@@ -1018,6 +1018,7 @@ static void priv_pcd_prepare(const struct device *dev)
 	priv->pcd.Init.ep0_mps = cfg->ep0_mps;
 	priv->pcd.Init.speed = PCD_SPEED_FULL;
 	priv->pcd.Instance = (PCD_TypeDef *)DT_INST_REG_ADDR(0);
+	priv->pcd.Init.phy_itface = DT_INST_PROP(0, phy_type);
```

Comment:
This change is not a refactoring, so for the very least, commit message should be updated.
Now, it is very interesting that this was not done on the driver. It is useless (and then maybe it is also useless in the other driver, and then it could just be removed) or was it actually missing here which would have been a bug ?

Reply:
Ok, I see that this commit was missing the matching code removal which is not added. Resolved.